products/urls.py

```diff
@@ -0,0 +1,18 @@
+from django.urls    import path
+from products.models import Category
+
+from products.views import (
+    CategoryView,
+    DetailProductView,
+    MenuView,
+    ProductView
+)
+
+urlpatterns = [
+    path('/menus',                               MenuView.as_view()),
+    path('/categories',                          CategoryView.as_view()),
+    path('/menus/<str:menu_name>',               CategoryView.as_view()),
+    path('',                                     ProductView.as_view()),
+    path('/<str:menu_name>/<str:category_name>', ProductView.as_view()),
+    path('/<int:id>',                            DetailProductView.as_view()),
```

Comment:
- 함수의 parameter 부분은 굳이 정렬 안해주셔도 괜찮습니다.
```suggestion
    path('/menus', MenuView.as_view()),
    path('/categories', CategoryView.as_view()),
    path('/menus/<str:menu_name>', CategoryView.as_view()),
    path('', ProductView.as_view()),
    path('/<str:menu_name>/<str:category_name>', ProductView.as_view()),
    path('/<int:id>', DetailProductView.as_view()),
```

Reply:
수정했습니다